src/fluree/db/json_ld/vocab.cljc

```diff
@@ -195,22 +196,26 @@
               "_default"   {:name "_default" :id 11 :sid nil}}
         pred (map-pred-id+iri [{:iri  "@id"
                                 :idx? true
-                                :id   0}
+                                :id   const/$iri}
                                {:iri  "@type"
                                 :ref? true
                                 :idx? true
-                                :id   200}
+                                :id   const/$rdf:type}
                                {:iri  "http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
                                 :ref? true
                                 :idx? true
-                                :id   200}])]
+                                :id   const/$rdf:type}
+                               {:iri "http://www.w3.org/2000/01/rdf-schema#Class"
+                                :ref? true
+                                :idx? true
+                                :id   const/$rdfs:Class}])]
     {:t           0
      :refs        #{}
      :coll        coll
      :pred        pred
      :context     nil
      :context-str nil
-     :shapes (atom {:class {}
+     :shapes (atom {:class {} ; TODO: Does this need to be an atom?
```

Comment:
@bplatz I came across this atom holding what looks like SHACL shapes, which means we no longer can treat a db as an immutable value, and I think this will lead to coordination and encapsulation problems related to keeping the right state updated for the right db objects at the right time. What does the atom buy us that an immutable map would not?

Reply:
@zonotope This is here as a shared cache - it could get moved to conn/ledger - would need to think through how to clean/expire that more centralized cache efficiently.

Reply:
Ok, that makes sense. I think we might be able to move it to the conn, and/or use the conn to manage creating/updating the cache as it loads and queries new, immutable dbs. I think that gives the most flexibility in terms of cache updates/evictions while still isolating mutable state and allowing us to treat dbs as flat data. (after alpha.x)

Reply:
Thinking about scaling a server processing an immense amount of operations across ledgers -- single atom will quickly become a point of resource contention. So even if held at the conn, I think utilizing atoms per ledger or maybe even at times per-db is still a good strategy.

In practice we need to cache DBs themselves as they will be used repeatedly across multiple threads, so I'd like to consider the data a DB handles is immutable, but that doesn't mean its implementation and optimization resources have to be... but like the idea of moving some of this off the db (and ledger's) plate when it makes sense.